### DISCUSSION TITLE
Can not preview the DDL sql after I add a new column

### DISCUSSION THREAD

**Discussion by SSSDNSY** [dbeaver/dbeaver#22914] [Q&A]
### Description

as the title descrition,Can not preview the DDL after I add a new column, the previous version (maybe version number < 23.2.* ) of dbever can preview,because I usually copy this DDL to backup.
![image](https://github.com/dbeaver/dbeaver/assets/15624361/763de9ee-f4b6-4c4b-9b55-14da91f614da)
1、double click on the any table of mysql in the sql Editor , it will show all columns of table
2、add new column or drop 
3、save (the previous version can show the preview DDL of this modify.

### DBeaver Version

 23.3.3.202401211839

### Operating System

win10

### Database and driver

mysql8

### Steps to reproduce

1、double click on the any table of mysql in the sql Editor , it will show all columns of table
2、add new column or drop 
3、save (the previous version can show the preview DDL of this modify.

**Comment by SSSDNSY** [comment]
1、double click on the any table of mysql in the sql Editor , it will show all columns of table
2、add new column or drop 
3、save (the previous version can show the preview DDL of this modify.

随便点开一张mysql的表，并编辑字段，点击保存时，之前的版本是有预览本次修改的DDL的功能，现在没有了，BUG?

**Comment by LonwoLonwo** [comment]
Hello @SSSDNSY 

Please check that this setting is enabled in your preferences:

![2024-02-21 11_58_00-Preferences](https://github.com/dbeaver/dbeaver/assets/45152336/d664792f-d036-49d2-8ae7-d927f74595bd)

**Comment by SSSDNSY** [comment]
![image](https://github.com/dbeaver/dbeaver/assets/15624361/a0d383e5-1cdb-4bf9-b5c4-a1134e5a6fd5)
oooh, sorry, it's not bug ,I can not find the setting  in my preferences if you dont tell me. thanks ! I'll close this issue.
